packages/documentation-site/patternfly-docs/content/PatternFly-AI/conversation-design/conversation-design.md

```diff
@@ -0,0 +1,75 @@
+---
+id: Conversation design
+section: PatternFly-AI
+---
+
+import { Button, Flex, FlexItem } from '@patternfly/react-core';
+import ArrowRightIcon from '@patternfly/react-icons/dist/esm/icons/arrow-right-icon';
+
+
+# Conversation design guidelines
+
+**Conversation design** is method of writing for conversational interfaces, like chatbots or voicebots. The goal of conversation design is to create an interactive experience that resembles human-to-human conversation as much as possible. Like traditional content design, conversation design is focused on using words to make experiences clear, concise, and well-timed.
+
+Good conversation design combines content strategy, writing, and design to ensure that the development and use of chatbots is centered on real user needs. This includes making sure that all text related to the conversational UI supports easy engagement with chatbots, while staying aligned with brand standards.
+
+When designing AI conversations alongside PatternFly-based projects, it is important to align with our established [brand voice and tone](/ux-writing/brand-voice-and-tone), as well as our [ethical guidelines for AI.](/patternfly-ai/ai-guidelines)
```

Comment:
"ethical guidelines for AI" link is broken.

Reply:
this page should be added by https://github.com/patternfly/patternfly-org/pull/4175! will probably be worth a quick double check on links once all the AI docs are merged to make sure they work